addon/templates/components/common/bs-collapse.hbs

```diff
@@ -0,0 +1,11 @@
+<div
+  class="{{if this.collapse "collapse"}} {{if this.collapsing "collapsing"}} {{if this.showContent (if (has-bootstrap-version 4) "show" "in")}}"
```

Comment:
I think we should come up with a convention around `{{has-bootstrap-version}}` helper. I have the feeling that using the else part is confusing with that helper and doesn't help with readability. Even so it would be more verbose I think using it multiple times might be easier to read:

```suggestion
  class="{{if this.collapse "collapse"}} {{if this.collapsing "collapsing"}} {{if this.showContent (if (has-bootstrap-version 4) "show") (if (has-bootstrap-version 3) "in")}}"
```

This would also require less refactoring and would be less error prune if another bootstrap version is added.